### PR TITLE
add MDPriority.IsValid

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: go/protocol/keybase1/extras.go [hunk 1]
```diff
@@ -1801,3 +1801,8 @@ const (
 	// remote helpers.
 	MDPriorityGit = 32
 )
+
+// IsValid returns true is p is a valid MDPriority, or false otherwise.
+func (p MDPriority) IsValid() bool {
+	return p < 256 && p >= 0
```
Comment (jzila) [inline go/protocol/keybase1/extras.go]: Is this going to be true even after the `is_placeholder` refactor?

Comment (songgao) [reply go/protocol/keybase1/extras.go]: Yup it used to be signed `TINYINT`, and not it's unsigned `TINYINT`.

**Comment by songgao** [comment]
It builds locally on my machine, so I'm merging without waiting for CI so I can vendor it in from master.

Will keep an eye on master build.

File: go/protocol/keybase1/extras.go (hunk 1 above)
Comment (jzila) [reply go/protocol/keybase1/extras.go]: But if it's unsigned, then isn't `p >= 0` tautalogical?

Comment (songgao) [reply go/protocol/keybase1/extras.go]: Yes, but this is typed as `int` generated by msgpack/avdl, which doesn't really support unsigned int. So we need these checks on server side before attempting to put it in to DB.

**Pull request merged**